Traceback (most recent call last):
  File "/home/michael/Documents/Python/errorExample2.py", line 7, in <module>
    raise Exception('This is an error message.')
Exception: This is an error message.
